components/server/src/ome/services/fulltext/FullTextBridge.java

```diff
@@ -356,6 +362,31 @@ private void handleFileAnnotation(final Document document,
     }
 
     /**
+     * Creates {@link Field} instances for {@link MapAnnotation} named-value
+     * pair.
+     *
+     * @param document
+     * @param store
+     * @param index
+     * @param boost
+     * @param mapAnnotation
+     */
+    private void handleMapAnnotation(final Document document,
+            final LuceneOptions opts, MapAnnotation mapAnnotation) {
+logger().error("handling map annotation");
+        List<NamedValue> nvs = mapAnnotation.getMapValue();
+        if (nvs != null && nvs.size() > 0) {
+            for (NamedValue nv : nvs) {
+                if (nv != null) {
+logger().error("{}={}", nv.getName(), nv.getValue());
+                    add(document, nv.getName(), nv.getValue(), opts);
```

Comment:
Happy to entertain concrete suggestions. My thinking was that even a collision would still return proper search results. A prefix will require users to know to add it. *Shrugs* Both perhaps?